app/connectors/SubmitVatReturnConnector.scala

```diff
@@ -41,10 +38,10 @@ class SubmitVatReturnConnector @Inject()(val http: HttpClient, val appConfig: Mi
         "Environment" -> appConfig.desEnvironment,
         "OriginatorID" -> originatorID
       )
-      .copy(authorization = Some(Authorization(s"Bearer ${appConfig.desToken}")))
+      .copy(authorization = None)
 
     Logger.debug(s"[SubmitVatReturnConnector][submitVatReturn] Submitting VAT Return to URL: ${desVatReturnsUrl(vrn)}. Body: ${Json.toJson(model)}")
-    Logger.debug(s"[SubmitVatReturnConnector][submitVatReturn] Headers: ${hc.headers}")
+    Logger.debug(s"[SubmitVatReturnConnector][submitVatReturn] Headers: ${hc.extraHeaders}")
     http.POST[VatReturnSubmission, HttpGetResult[SuccessModel]](desVatReturnsUrl(vrn), model)(
```

Comment:
As mentioned above, you now need to pass HTTP headers into this function explicitly so that calls to external services (DES) can see them.